flink/v1.19/flink/src/test/java/org/apache/iceberg/flink/source/TestFlinkInputFormatReaderDeletes.java

```diff
@@ -35,6 +35,7 @@
 import org.apache.iceberg.util.StructLikeSet;
 
 public class TestFlinkInputFormatReaderDeletes extends TestFlinkReaderDeletesBase {
+
```

Comment:
I think it would be good to remove all of these whitespace changes, which would make it much easier to review the PR

Reply:
please see my comment above